packages/pluggableWidgets/rich-text-web/src/ui/RichText.scss

```diff
@@ -0,0 +1,40 @@
+@use "../../../node_modules/ckeditor4/contents.css";
```

Comment:
`@use "~ckeditor4/contents.css";` will automatically resolve in the correct node_modules without relative path.

Reply:
Same applies for lines below